### ISSUE TITLE
[Feature Request] Expose an HA-facing configuration option to pass environment variables

### ISSUE THREAD

**Issue by LanceMcCarthy** [hassio-addons/addon-bitwarden#229]
# Problem/Motivation

I would like to be able to set two environment variables, particularly:

- `globalSettings__disableUserRegistration` (which is `SIGNUPS_ALLOWED` in Vaultwarden)
- SMTP settings

I imagine it would be a nightmare to ask for this add-on's maintainers to all these settings individually. It's also dangerous to allow for a blanket list of environment variables to be passed in a single input.

- Docs on configuration here => https://github.com/dani-garcia/vaultwarden/wiki/Configuration-overview
- The default env vars used are here => https://github.com/dani-garcia/vaultwarden/blob/main/.env.template

## Work Proposal - PR

In preparation for opening this Issue, I did my due diligence and studied how this add on is built, as well as how `vaultwarden/server`  new user registration setting works https://github.com/dani-garcia/vaultwarden/wiki/Disable-registration-of-new-users 

**If it is agreeable** that these are nice-to-have capabilities, I will do the work and add the `allow_signups: true` option to this list

https://github.com/hassio-addons/addon-bitwarden/blob/0b62b598554087ed20f8efcbc54e75311a05f82b/bitwarden/config.yaml#L20-L23

~~I'm not yet sure how to pass variables from the options list to the Dockerfile yet, but I'll figure that out. At first glance, I imagine I can then add the `ENV SIGNUPS_ALLOWED=${allow_signups}` ENV line for here, but I'll need to test and debug~~

**[EDIT]**
Upon further inspection, I think I can just set it in the `bitwarden/run` startup script with:

```bash
export SIGNUPS_ALLOWED=${allow_signups}
```

right before:

https://github.com/hassio-addons/addon-bitwarden/blob/0b62b598554087ed20f8efcbc54e75311a05f82b/bitwarden/rootfs/etc/services.d/bitwarden/run#L74-L81

### Future STMP Settings

If this proposal is accepted, then everything goes to plan and my PR is merged, I'll continue the work to add the SMTP options in a future update after proving myself with the simple bool option.

**Comment by frenck** [comment]
Just go to the admin interface (`/admin`) and set all of those options. No need to duplicate those into the HA interface or use env vars for that stuff. 👍 

../Frenck

**Comment by LanceMcCarthy** [comment]
<facepalm> it's been a while since I've set this up and forgot about it, thank you for the reminder... and sorry for replying in a closed thread (I know it's annoying)

It looks like the admin panel is no longer accessible for me:

![image](https://user-images.githubusercontent.com/3520532/208507590-ea486b0f-a76a-41cc-871b-6a65902c396d.png)

Has this behavior changed, or do I need to delete the add-on and start over?

note: I still have my admin token (ironically stored in LastPass 🤣), I tried passing it through HTTP header and querystring, but neither work. I think they need the env var to be set.
